.github/workflows/holiday-generator.yml

```diff
@@ -0,0 +1,31 @@
+name: Generate holiday ICS files
+
+on:
+    schedule:
+        # Run every year on 1st December, midnight
+        - cron: '0 0 1 12 *'
+    workflow_dispatch:
+
+jobs:
+    generate:
+        name: Generate ICS
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@master
```

Comment:
Shouldn't we use `actions/checkout@v4`? 

The [`master`](https://github.com/actions/checkout/tree/master) branch doesn't seem to be updated anymore as the last commit was 4 years ago. Or am I missing something here?

Reply:
Well, the only reason why I've used `master` is because I've copied it from another of my GitHub Actions 😄. I think I can safely change it to `v4`, there shouldn't be any breaking changes, especially that its only purpose is to check out the repo.